app/views/enrollment_mailer/sent_application_notification.text.erb

```diff
@@ -0,0 +1,5 @@
+Notre service juridique va prendre en charge votre demande au plus vite. Nous répondons habituellement sous 2 semaines, mais cela peut prendre plus longtemps lorsque nous avons beaucoup de demandes.
+
+Si vous avez besoin de plus d'information vous pouvez répondre à cet email directement.
```

Comment:
To my opinion, there should be a mention here of "please use our public test environment while we are analysing your demand"